tfe/data_source_workspace_ids.go

```diff
@@ -55,14 +68,26 @@ func dataSourceTFEWorkspaceIDsRead(d *schema.ResourceData, meta interface{}) err
 	ids := make(map[string]string, len(names))
 
 	options := tfe.WorkspaceListOptions{}
+
+	// Create a search string with all the tags we are looking for.
+	var tagSearchParts []string
+	for _, tagName := range d.Get("tag_names").([]interface{}) {
+		id += tagName.(string) // add to the state id
```

Comment:
thoughts on adding a check to see if `tagName` is empty?

```
name := tagName.(string)
if name != "" {
  id += name
  tagSearchParts = append(tagSearchParts, name)
}
```

Reply:
that seems like it would be caused by an error elsewhere but i could see a variable being used here and it ending up blank. 👍🏻